lib/models/file.dart

```diff
@@ -224,7 +224,7 @@ class File extends EnteFile {
     }
   }
 
-  String getDisplayName() {
+  String get displayName {
```

Comment:
this can be null because title can be null

Reply:
Have made sure that it doesn't return null
https://github.com/ente-io/frame/blob/f3213cc848ae6bdc447d0e4ce4e67f2740ffaf9f/lib/models/file.dart#L232